File: cs193p/Scroller/Scroller/ViewController.m
//
//  ViewController.m
//  Scroller
//
//  Created by mcxiaoke on 15/8/18.
//  Copyright (c) 2015年 mcxiaoke. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIScrollView *scroller;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.scroller.contentSize=CGSizeMake(500, 1000);
}



@end

File: osx_learning_demos/BabyScript/BabyScript/ViewController.swift
//
//  ViewController.swift
//  BabyScript
//
//  Created by mcxiaoke on 16/6/28.
//  Copyright © 2016年 mcxiaoke. All rights reserved.
//

import Cocoa

class ViewController: NSViewController {
  
  @IBOutlet var text:NSTextView!
  
  @IBAction func showWordCountWindow(sender:AnyObject){
    let storyboard = NSStoryboard(name: "Main", bundle: nil)
    let wordCountWindowController = storyboard.instantiateControllerWithIdentifier("WordCountWindowController") as! NSWindowController
    if  let wordCountWindow = wordCountWindowController.window, textStorage = text.textStorage {
      let wordCountViewController = wordCountWindow.contentViewController as! WordCountViewController
      wordCountViewController.wordCount.stringValue = "\(textStorage.words.count)"
      wordCountViewController.paragraphCount.stringValue = "\(textStorage.paragraphs.count)"
      NSApplication.sharedApplication().runModalForWindow(wordCountWindow)
    }
  }
  
  override func viewDidLoad() {
    super.viewDidLoad()
    text.toggleRuler(nil)
  }

}

